Makefile

```diff
@@ -224,8 +224,10 @@ endif
 ### manifests: Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=role webhook paths="./..." \
+			crd:crdVersions=v1beta1 \
 			output:crd:artifacts:config=deploy/templates/crd/bases \
 			output:rbac:artifacts:config=deploy/templates/components/rbac
+	rm -rf ./config
```

Comment:
controller-gen creates v1 at config folder, I did not figure out how to disable it, so just came to removing this file.